forte/sci/fci_mo.cc

```diff
@@ -343,16 +343,16 @@ std::vector<double> FCI_MO::compute_ss_energies() {
     }
     print_CI(nroot_, options_->get_double("FCIMO_PRINT_CIVEC"), eigen_, determinant_);
 
-//    if (integral_->integral_type() != Custom) {
```

Comment:
@lcyyork : do you think we still need this code?

Reply:
No, they are not needed.

Reply:
deleted